libcrux-ml-kem/src/lib.rs

```diff
@@ -51,16 +74,104 @@
+//!   encapsulation.
+//! - `pqcp`: See above.
+//! - `codec`: Use this feature to enable (de-)serialization of (packed)
+//!   public keys and ciphertexts using
+//!   [`tls_codec`](https://crates.io/crates/tls_codec).
+//!
+//! ## Secret Independence
+//! As outlined in the description of the `check-secret-independence`
+//! feature above, we leverage the Rust type system to ensure that secret
+//! values are not used in operations that are known to have
+//! data-dependent execution time. While the implementation of constant time
+//! operations is best-effort, as there are no guarantees from the
+//! compiler, we follow established constant-time patterns and validate
+//! constant-time code via inspection of the generated assembly
+//! instructions.
+//!
+//! ## Verification
+//! ![verified]
 //!
-//! ## Features
+//! The portable and AVX2 code for field arithmetic, NTT polynomial arithmetic, serialization, and the generic code for high-level algorithms
+//! is formally verified using [hax](https://cryspen.com/hax) and  [F*](https://fstar-lang.org).
 //!
-//! By default, all ML-KEM parameter sets are enabled. If required, they are
-//! available individually under feature flags `mlkem512`, `mlkem768`,
-//! `mlkem1024`.
+//! Please refer to [this file](proofs/verification_status.md) for detail on the verification of this crate.
 //!
-//! ### Kyber Round 3
-//! The `kyber` flag also gives access to an, as yet, unverified implementation
-//! of Kyber as submitted in Round 3 of the NIST PQ competition.
+//! [verified]: https://img.shields.io/badge/verified-brightgreen.svg?style=for-the-badge&logo=data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0idXRmLTgiPz48IS0tIFVwbG9hZGVkIHRvOiBTVkcgUmVwbywgd3d3LnN2Z3JlcG8uY29tLCBHZW5lcmF0b3I6IFNWRyBSZXBvIE1peGVyIFRvb2xzIC0tPg0KPHN2ZyB3aWR0aD0iODAwcHgiIGhlaWdodD0iODAwcHgiIHZpZXdCb3g9IjAgMCAyNCAyNCIgZmlsbD0ibm9uZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4NCjxwYXRoIGQ9Ik05IDEyTDExIDE0TDE1IDkuOTk5OTlNMjAgMTJDMjAgMTYuNDYxMSAxNC41NCAxOS42OTM3IDEyLjY0MTQgMjAuNjgzQzEyLjQzNjEgMjAuNzkgMTIuMzMzNCAyMC44NDM1IDEyLjE5MSAyMC44NzEyQzEyLjA4IDIwLjg5MjggMTEuOTIgMjAuODkyOCAxMS44MDkgMjAuODcxMkMxMS42NjY2IDIwLjg0MzUgMTEuNTYzOSAyMC43OSAxMS4zNTg2IDIwLjY4M0M5LjQ1OTk2IDE5LjY5MzcgNCAxNi40NjExIDQgMTJWOC4yMTc1OUM0IDcuNDE4MDggNCA3LjAxODMzIDQuMTMwNzYgNi42NzQ3QzQuMjQ2MjcgNi4zNzExMyA0LjQzMzk4IDYuMTAwMjcgNC42Nzc2NiA1Ljg4NTUyQzQuOTUzNSA1LjY0MjQzIDUuMzI3OCA1LjUwMjA3IDYuMDc2NCA1LjIyMTM0TDExLjQzODIgMy4yMTA2N0MxMS42NDYxIDMuMTMyNzEgMTEuNzUgMy4wOTM3MyAxMS44NTcgMy4wNzgyN0MxMS45NTE4IDMuMDY0NTcgMTIuMDQ4MiAzLjA2NDU3IDEyLjE0MyAzLjA3ODI3QzEyLjI1IDMuMDkzNzMgMTIuMzUzOSAzLjEzMjcxIDEyLjU2MTggMy4yMTA2N0wxNy45MjM2IDUuMjIxMzRDMTguNjcyMiA1LjUwMjA3IDE5LjA0NjUgNS42NDI0MyAxOS4zMjIzIDUuODg1NTJDMTkuNTY2IDYuMTAwMjcgMTkuNzUzNyA2LjM3MTEzIDE5Ljg2OTIgNi42NzQ3QzIwIDcuMDE4MzMgMjAgNy40MTgwOCAyMCA4LjIxNzU5VjEyWiIgc3Ryb2tlPSIjMDAwMDAwIiBzdHJva2Utd2lkdGg9IjIiIHN0cm9rZS1saW5lY2FwPSJyb3VuZCIgc3Ryb2tlLWxpbmVqb2luPSJyb3VuZCIvPg0KPC9zdmc+
```

Comment:
Having this everywhere adds lots of blob. We should add these labels to the repo at some point and just link to it.

Anyway, not for this PR. Just a thought.